pdblp/pdblp.py

```diff
@@ -780,9 +780,9 @@ def secf(self, query, max_results=10, yk_filter="NONE"):
                 ticker = ticker.replace('<', ' ').replace('>', '').upper()
                 descr = r.getElementAsString("description")
                 if not descr.endswith('(Multiple Matches)'):
-                    data.append(ticker)
-        return data
-        
+                    data.append((ticker, descr))
+        return pd.DataFrame(data, columns=['ticker', 'description'])
```

Comment:
Given that the response messages look like
```
InstrumentListResponse = {
    results[] = {
        results = {
            security = "Some security"
            Description = "Blah blah"
        }
   ...
    }
}
```
I think that `columns = ['security', 'description']` would be more appropriate